### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (Exception has been thrown by the target of an invocation.))

### DISCUSSION THREAD

**Discussion by refleraga** [ppy/osu#16724] [Q&A] [answered]
[attach_me.zip](https://reflega.s-ul.eu/yY2u2XtD) (github wouldn't let me upload directly for some reason)

**Comment by frenzibyte** [comment, marked as answer]
The attached EF database migrates successfully on my end. Can you attach the game logs if you haven't tried to run the game a second time (see #15188)? The exception message in title is not quite clear (perhaps we should consider attaching the full stack trace in the description).

Here's a fixed version of your database that you can overwrite the existing one with: [client.realm.zip](https://www.dropbox.com/s/449keil0cyp42ih/client.realm.zip?dl=0)

**Reply by refleraga** [reply]
thanks! unfortunately the migration happened days ago, and i've run the game a few times since so i don't have any logs from back then. i couldn't upload my client.db immediately because of my horrendous upload speed, and just didn't think of checking the logs at the time.

interestingly enough i had lazer do the migration again using the backups and no errors popped up.
